homeassistant/components/lifx/light.py

```diff
@@ -449,8 +449,31 @@ def __init__(self, bulb, effects_conductor):
         self.bulb = bulb
         self.effects_conductor = effects_conductor
         self.registered = True
+        self._attr_available = True
         self.postponed_update = None
         self.lock = asyncio.Lock()
+        self._attr_unique_id = self.bulb.mac_addr
+        self._attr_name = self.bulb.label
+        """set the coldest/warmest color_temp that this light supports."""
```

Comment:
```suggestion
        
        # Set the coldest/warmest color_temp that this light supports.
```

Reply:
Done. 

I think there are nothing to adjust in the subclasses. I could convert "supported_features" in LIFXColor, but that would demand an __init__ method, so we would end up with the same amount of code.

Reply:
This is static and should be defined as a class attribute with `_attr_effect_list = [SERVICE_EFFECT_PULSE, SERVICE_EFFECT_STOP]`
https://github.com/home-assistant/core/blob/848c0be58a45371cb2b9a03398075ef00d136331/homeassistant/components/lifx/light.py#L658-L661


This is static and should be defined as a class attribute with `_attr_effect_list = [[SERVICE_EFFECT_COLORLOOP, SERVICE_EFFECT_PULSE, SERVICE_EFFECT_STOP] `
https://github.com/home-assistant/core/blob/848c0be58a45371cb2b9a03398075ef00d136331/homeassistant/components/lifx/light.py#L674-L677

None of which will require an init, and will actually save you lines.

Reply:
My bad, I did a "git grep" but I surely have misspelled _attr_effect_list because it came up empty handed.

Anyhow corrected, thanks for catching this.

Reply:
Great :+1:  now we just need to add coverage for these changes or someone to verify the changes that use the integration :+1:

Reply:
Without a codeowner and without merging this PR, getting it tested is going to be a real challenge.

I can activate coverage and add tests, but that will be in a followup PR.

Reply:
we cannot merge it untested in this case.